charts/hush-sensor/templates/_helpers.tpl

```diff
@@ -429,3 +429,23 @@ that is also easy to retrieve with Helm.
 {{- $ks := lookup "v1" "Namespace" "" "kube-system" -}}
 {{- and $ks.metadata $ks.metadata.uid -}}
 {{- end }}
+
+{{/*
+Sentry service account annotations with AWS IAM role handling
+*/}}
+{{- define "hush-sensor.sentryServiceAccountAnnotations" -}}
+{{- $annotations := deepCopy (default dict .Values.sentry.serviceAccount.annotations) -}}
+{{- if and .Values.sentry.creds .Values.sentry.creds.aws .Values.sentry.creds.aws.irsa -}}
+  {{- if hasKey $annotations "eks.amazonaws.com/role-arn" -}}
```

Comment:
Consider defining `"eks.amazonaws.com/role-arn"` once. It is copied 4 times now.